### PR TITLE
Closes #767 - Distinguish between source and target shapes

### LINKED ISSUE #767: Distinguish between Source and Target shapes
An *source* shape is one that is received directly from some source system.
A *target* shape is one that is derived from other shapes via some transform.

I want the ability to specify that a given Shape is either a *source* shape or a *target* shape so that I can communicate this information to other people and to tools that will generate transforms for the target shapes.

### Acceptance Criteria

* The Konig core vocabulary must include OWL classes named `konig:SourceShape` and `konig:TargetShape`.
* It must be possible to specify the  `rdf:type` of a Shape using a spreadsheet that has a column named `Shape Type`.    For instance, one might have the following rows in a spreadsheet.

| Shape Id                              | Target Class         | Shape Type              | Datasource       |   
|-------------------------------|--------------------|-----------------------|------------------|
| shape:SourceProductShape | schema:Product  |  konig:SourceShape | aws:S3Bucket    |
| shape:TargetProductShape  | schema:Product  | konig:TargetShape   | aws:AwsAurora  |

### Implementation Notes
* You will need to enhance `WorkbookLoader` to support a new column named `Shape Type`.  See [How to enhance WorkbookLoader](https://github.com/konigio/konig/wiki/How-to-enhance-WorkbookLoader)

### PR DESCRIPTION
**Pull request by RanjaniGuru**
Closes #767

### CODE REVIEW

File: konig-core/src/main/java/io/konig/core/vocab/Konig.java [hunk 1]
```diff
@@ -67,7 +67,8 @@
 	// Generic Data Modeling
 
 	public static final URI id = new URIImpl("http://www.konig.io/ns/core/id");
-	public static final URI mediaTypeBaseName = new URIImpl("http://www.konig.io/ns/core/mediaTypeBaseName");
+	public static final URI mediaTypeBaseName = new URIImpl("http://www.konig.io/ns/core/mediaTypeBaseName");
+	public static final URI shapeType = new URIImpl("http://www.konig.io/ns/core/shapeType");
```
Comment (gmcfall) [inline konig-core/src/main/java/io/konig/core/vocab/Konig.java]: We should not mint the predicate `konig:shapeType`. Use the standard predicate `rdf:type`.

File: konig-core/src/main/java/io/konig/shacl/Shape.java [hunk 2]
```diff
@@ -88,7 +88,8 @@
 
 	private URI aggregationOf;
 	private URI rollUpBy;
-	private String mediaTypeBaseName;
+	private String mediaTypeBaseName;
+	private URI shapeType;
```
Comment (gmcfall) [inline konig-core/src/main/java/io/konig/shacl/Shape.java]: Remove the `shapeType` field.  Use the exisiting `List<URI> type` field instead.  (A Shape may have more than one `type` value.)

File: konig-spreadsheet/src/main/java/io/konig/spreadsheet/WorkbookLoader.java [hunk 3]
```diff
@@ -1791,6 +1794,7 @@ private void loadShapeRow(Row row) throws SpreadsheetException {
 			edge(shapeId, Konig.rollUpBy, rollUpBy);
 			edge(shapeId, Konig.mediaTypeBaseName, mediaType);
 			edge(shapeId, Konig.bigQueryTableId, bigqueryTable);
+			edge(shapeId, Konig.shapeType, shapeType);
```
Comment (gmcfall) [inline konig-spreadsheet/src/main/java/io/konig/spreadsheet/WorkbookLoader.java]: Replace with...
```
   edge(shapeId, RDF.TYPE, shapeType);
```

**Comment by RanjaniGuru** [comment]
Committed the code for #758 wrongly in ranjaniguru/#757 branch. Will fix the review comments and create a new pull request for the same.

**Pull request closed without merging**